src/app/components/Document/index.test.jsx

```diff
@@ -30,17 +31,22 @@ describe('Document', () => {
   };
   const styleTags = <style>{'html { color: red; }'}</style>;
 
-  const shallowDocument = shallowRender(
-    <Document
-      assets={assets}
-      app={'<h1>App!</h1>'}
-      data={data}
-      helmet={helmet}
-      styleTags={styleTags}
-    />,
-  );
+  const shallowDocument = isAmp =>
+    shallowRender(
+      <Document
+        assets={assets}
+        app={'<h1>App!</h1>'}
+        data={isAmp ? ampData : data}
+        helmet={helmet}
+        styleTags={styleTags}
+      />,
+    );
 
   it('should render correctly', () => {
-    expect(shallowDocument).toMatchSnapshot();
+    expect(shallowDocument(false)).toMatchSnapshot();
+  });
+
+  it('should render AMP version correctly', () => {
+    expect(shallowDocument(true)).toMatchSnapshot();
```

Comment:
Generally better practise to use an object - more readable (and extensible):

```js
expect(shallowDocument({ isAmp: true })).toMatchSnapshot();
```

Reply:
Thanks very much 🙂